modules/@apostrophecms/migration/index.js

```diff
@@ -245,8 +242,13 @@ module.exports = {
           // Just in case the db has no documents but did
           // start to run migrations on a previous attempt,
           // which causes an occasional unique key error if not
-          // corrected for here
-          await self.db.removeMany({});
+          // corrected for here.
+          //
+          // Other migration-related facts that are not migration
+          // names are stored with a leading *, leave them alone
+          await self.db.removeMany({
+            _id: /^[^*]/
```

Comment:
Somewhere to keep memos like the list of existing schema fields. Otherwise we have a proliferation of tiny mongo collections.

Reply:
We could have just one new, separate collection like `aposModuleSchema` or `aposRegisteredSchema` so we don't store it into the `aposMigration` collection, which is a bit tied to migrations, but not really...

Inside that dedicated collection, we could have the schema stored in documents: 1 document in that collection = 1 module schema.

That would require changing a lot of code, and since you've already finished the feature, maybe it's not worth it.
Just wanted to have your thoughts on that, to have the schemas stored in a dedicated collection for clarity